src/components/workers-page/workers-tab/workers-tab.component.tsx

```diff
@@ -52,10 +52,10 @@ export default function WorkersTab(): JSX.Element {
                 <TableCell align="left">{WorkerTypeHelper.translate(worker.type)}</TableCell>
                 <TableCell align="left">{worker.time}</TableCell>
                 <TableCell align="right">
-                  <Button>Edytuj</Button>
+                  <Button variant="secondary">Edytuj</Button>
                 </TableCell>
                 <TableCell align="right">
-                  <Button>Usuń</Button>
+                  <Button variant="secondary">Usuń</Button>
```

Comment:
Pozmieniałem wszędzie przyciski na nasze, żeby kiedy @Ehevi  dogra style to wszystko odrazu się zaplikowało